style.css

```diff
@@ -278,3 +282,205 @@ footer {
   gap: 20px;
   font-size: 22px;
 }
+
+.hide-mobile {
+  display: none;
+}
+
+.hide-desktop {
+  display: block;
+}
+
+.fa-envelope {
+  display: none !important;
+}
+
+@media screen and (min-width: 768px) {
+  .fa-bars {
+    display: none !important;
+  }
+
+  .fa-envelope {
+    display: block !important;
+  }
+
+  .hide-mobile {
+    display: block;
+  }
+
+  .hide-desktop {
+    display: none;
+  }
+
+  nav {
+    display: flex;
+    gap: 20px;
+  }
+
+  header {
+    padding: 20px 70px;
+  }
+
+  .landing {
+    height: 90vh;
+    display: flex;
+    flex-direction: row;
+
+    /* justify-content: space-around; */
+    padding: 200px 50px;
+    background-color: #1c1a19;
+    color: white;
+    background-image: url("resources/Illustration-header.png");
+    background-repeat: no-repeat;
+    background-position: right;
+  }
+
+  .icons {
+    display: flex;
+    flex-direction: column;
+    margin-top: 80px;
+    margin-right: 50px;
+    gap: 10px;
+    font-size: 25px;
+  }
+
+  .hey {
+    width: 500px;
+  }
+
+  h1 {
+    color: #ff6b00;
+    font-size: 72px;
+    font-family: "Crete Round", serif;
+  }
+
+  .roboto {
+    font-family: "Roboto", sans-serif;
+    font-size: 32px;
+  }
+
+  .description {
+    font-family: "Poppins", sans-serif;
+    font-size: 20px;
+  }
+
+  .sec2 {
+    position: relative;
+  }
+
+  .underline2 {
+    position: absolute;
+    left: 100%;
+    bottom: 50%;
+    width: 750px;
+  }
+
+  .card1 {
+    display: flex;
+    gap: 10;
+  }
+
+  .sec3 {
+    padding: 0 200px;
+  }
+
+  .card1 > img {
+    width: 500px;
+  }
+
+  .card1-text {
+    margin-left: 15px;
+    width: 500px;
+
+    /* margin-right: 70px; */
+  }
+
+  .card2-parent {
+    display: flex;
+    flex-wrap: wrap;
+    gap: 15px;
+  }
+
+  .about {
+    background-color: #1c1a19;
+    background-image: url("resources/Frame (1).png");
+    background-repeat: no-repeat;
+    background-position: right;
+    height: 90vh;
+    padding: 30px 180px;
+    margin-top: 120px;
+  }
+
+  .about > p {
+    color: #fff;
+    margin: 20px 0;
+    font-family: "Inter", sans-serif;
+    width: 500px;
+  }
```

Comment:
- [ ] Please make sure this text breaks into the next line at the same point where it breaks in the Figma design